pandas/tests/indexing/test_floats.py

```diff
@@ -53,8 +53,8 @@ def test_scalar_error(self, index_func):
             s.iloc[3.0]
 
         msg = (
-            "cannot do positional indexing on {klass} with these "
-            r"indexers \[3\.0\] of type float".format(klass=type(i).__name__)
+            fr"cannot do positional indexing on {type(i).__name__} with these "
```

Comment:
the r prefix not needed.

Reply:
I still have some PRs pending to submit, will fix on one of those